ChangeLog

```diff
@@ -102,6 +102,10 @@ What's New in Pylint 1.8?
 
       Close #574
 
+    * Check for string concatenation inside a list, introducing new
```

Comment:
It's a lie, you do not check list, but also tuples.

What about set literals? To be consistent, I'd expect to see lists, tuples and sets supported

Reply:
Good idea, fixed